com.unity.render-pipelines.universal/Shaders/PostProcessing/FinalPost.shader

```diff
@@ -5,18 +5,19 @@ Shader "Hidden/Universal Render Pipeline/FinalPost"
         #pragma multi_compile_local _ _FXAA
         #pragma multi_compile_local _ _FILM_GRAIN
         #pragma multi_compile_local _ _DITHERING
-		#pragma multi_compile_local _ _LINEAR_TO_SRGB_CONVERSION
-        
+        #pragma multi_compile_local _ _LINEAR_TO_SRGB_CONVERSION
+        #pragma multi_compile _ _USE_DRAW_PROCEDURAL
```

Comment:
Adding more "multi_compile _ " will only increase shader build times. Postprocessing is fillrate intensive not vertex bound, and there's not much advantage in draw procedural vs rendering a quad. I would remove the extra multi_compile _ and use quad.

Reply:
URP team plan to migrate to use procedural as well. I'd say it's fine for now to take the extra compilation variants until we handle this. @thomas-zeng we should at the very least have `multi_compile_local`